melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/mottak/SedMottakService.java

```diff
@@ -41,6 +45,10 @@ public void behandleSed(SedMottattHendelse sedMottattHendelse) {
             return;
         }
 
+        if (unleash.isEnabled("melosys.eessi.sed.rekkefølge") && sedMottattHendelse.getSedHendelse().erXSed() && !journalpostSedKoblingService.erASedAlleredeBehandlet(sedMottattHendelse.getSedHendelse().getRinaSakId())) {
```

Comment:
Har vi ToggleName eller noe tilsvarende i eessi? (Trengs sikkert ikke, men jeg spør i tilfelle ettersom vi bestemte oss for å bruke det i api)

Reply:
En anbefaling kan være å wrappe conditionals i metode eller variabel når det er flere enn en for å gjøre det lettere å lese koden

if(erXSedAlleredeBehandletUtenASed)?

Reply:
hva mener du med toggleName?

Skal wrappe 👍